napari_plugin_manager/qt_plugin_dialog.py

```diff
@@ -437,15 +437,16 @@ def _count_visible(self) -> int:
     def addItem(
         self,
         project_info_versions: Tuple[
-            npe2.PackageMetadata, List[str], List[str]
+            npe2.PackageMetadata, str, List[str], List[str]
```

Comment:
It looks like we should use `NamedTuple` here

Reply:
As this code was meant to be "temporary", I think we never went the named tuple direction.

It could help indeed :) . Do you want me to include that on this PR?

Reply:
But this temporary code tries to survive. So I think it is worth to use Namedtuple for better readability.